FetchMigration/Dockerfile

```diff
@@ -5,7 +5,7 @@ COPY python/Pipfile python/Pipfile.lock ./
 RUN apt -y update
 RUN apt -y install python3 python3-pip
 RUN pip3 install pipenv
-RUN pipenv install --system --deploy --ignore-pipfile
```

Comment:
This is included because I was getting this error while deploying to aws. It seems odd that I would run into this and no one else would, so it's possible there's another explanation.
cc: @peternied 

```
#10 [ 6/10] RUN pipenv install --system --deploy --ignore-pipfile
#10 0.471 Usage: pipenv install [OPTIONS] [PACKAGES]...
#10 0.471
#10 0.471 ERROR:: --system is intended to be used for Pipfile installation, not installation of specific packages. Aborting.
#10 0.471 See also: --deploy flag.
#10 ERROR: process "/bin/sh -c pipenv install --system --deploy --ignore-pipfile" did not complete successfully: exit code: 2
------
 > [ 6/10] RUN pipenv install --system --deploy --ignore-pipfile:
0.471 Usage: pipenv install [OPTIONS] [PACKAGES]...
0.471
0.471 ERROR:: --system is intended to be used for Pipfile installation, not installation of specific packages. Aborting.
0.471 See also: --deploy flag.
------
Dockerfile:8
--------------------
   6 |     RUN apt -y install python3 python3-pip
   7 |     RUN pip3 install pipenv
   8 | >>> RUN pipenv install --system --deploy --ignore-pipfile
   9 |
  10 |
--------------------
ERROR: failed to solve: process "/bin/sh -c pipenv install --system --deploy --ignore-pipfile" did not complete successfully: exit code: 2
```

Reply:
Did you have this commit when you got that error? https://github.com/opensearch-project/opensearch-migrations/pull/753